Assets/Prefabs/Surface_Entities/pCube17.prefab

```diff
@@ -150,5 +150,5 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   homePlanet: {fileID: 0}
   planets: []
-  ArbitraryVelocityDeltaThreshold: 1
-  VelocityDeltaCatchupRate: 0.4
+  ArbitraryVelocityDeltaThreshold: 5
+  VelocityDeltaCatchupRate: 0.5
```

Comment:
Go ahead and revert the changes here as you didn't see good results with the test.